casper/src/main/scala/coop/rchain/casper/genesis/contracts/Validator.scala

```diff
@@ -0,0 +1,26 @@
+package coop.rchain.casper.genesis.contracts
+
+import coop.rchain.crypto.PublicKey
+import coop.rchain.models.Par
+import coop.rchain.rholang.build.CompiledRholangSource
+import coop.rchain.rholang.interpreter.ParBuilder
+import monix.eval.Coeval
+
+// TODO: Eliminate public key argument if unnecessary
+final case class Validator(pk: PublicKey, stake: Long) extends CompiledRholangSource {
```

Comment:
This commit says `Extract Validator data type into separate file.`

Yet, it introduces a new `Validator.scala` and removes `VaultUtils.scala`. Bad merge?